jenkins/job-configs/kubernetes-jenkins/node-e2e.yaml

```diff
@@ -17,7 +17,8 @@
         - build-discarder:
             num-to-keep: 200
     builders:
-         - docker-build-publish:
+        - activate-gce-service-account
+        - docker-build-publish:
```

Comment:
@timstclair @piosz do we need to add the kubekins@ service account anywhere for this to keep working?